src/os/bluestore/ZonedAllocator.cc

```diff
@@ -27,13 +28,12 @@ ZonedAllocator::ZonedAllocator(CephContext* cct,
       size(size),
       block_size((block_size & 0x00000000ffffffff)),
       zone_size(((block_size & 0x0000ffff00000000) >> 32) * 1024 * 1024),
-      starting_zone((block_size & 0xffff000000000000) >> 48),
-      nr_zones(size / zone_size),
-      write_pointers(nr_zones) {
+      starting_zone_num((block_size & 0xffff000000000000) >> 48),
```

Comment:
leave a comment about blocksize piggybacking?

Reply:
Done.  Thanks for the review and the approval!